src/applications/vaos/components/FacilityAddress.jsx

```diff
@@ -17,14 +17,31 @@ export default function FacilityAddress({
           <br />
         </>
       )}
-      {address?.line?.map(line => (
-        <React.Fragment key={line}>
-          {line}
+      {!!address && (
+        <>
+          {address?.line?.map(line => (
+            <React.Fragment key={line}>
+              {line}
+              <br />
+            </React.Fragment>
+          ))}
+          {address.city}, {address.state} {address.postalCode}
+          <br />
+        </>
+      )}
+      {!address && (
+        <>
+          This appointment is scheduled with a community care provider. Please
```

Comment:
This `FacilityAddress` component is used in a few different places, not always just on the confirmed appointments card. So I'm not sure we should put this language in here. I think it might make more sense in `ConfirmedCommunityCareLocation`, where we could put some logic that renders `FacilityAddress` if we have an address, and this text if we do not.

Reply:
That makes perfect sense. The alert text is now placed in the `communityCareInstructions` component